### ISSUE TITLE
adding a participant - problem

### ISSUE THREAD

**Issue by radupoleuca** [academind/django-practical-guide-course-code#4]
When I add a participant (his email address) using the form, the email address is added to all meetings, not just the one he joined.
I followed the whole tutorial and wrote the fixed code as well. Has anyone fixed this bug?

**Comment by KyaUJY** [comment]
Yes, I'm having the same problem with the code I reproduced from the video, but it's also true for the code in this repository.
I still don't know what the problem is.
Thanks for this great content and class.

> When I add a participant (his email address) using the form, the email address is added to all meetings, not just the one he joined.
> I followed the whole tutorial and wrote the fixed code as well. Has anyone fixed this bug?
